src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -465,7 +469,16 @@ def perform_update(self, serializer):
             zaaktype=str(updated_zaak.zaaktype),
             partial=serializer.partial,
         )
-        send_zaak_cloudevent(ZAAK_GEMUTEERD, updated_zaak, self.request)
+
+        # TODO validate that this is the only correct situation to send this event for
```

Comment:
This is something I need to check with Joeri/Logius to see if this is the way MijnOverheid will update laatstGeopend (PATCH with only that attribute), so leaving this TODO here until we know for sure that this is the way it will be done